scripts/post-build.js

```diff
@@ -0,0 +1,4 @@
+const fs = require('fs');
+
+const { scripts, devDependencies, husky, ...cleanPackage } = JSON.parse(fs.readFileSync('package.json').toString());
```

Comment:
Nice 👍  - this would also be kinda cool to have as npm package. Basically that you could type something like `build-package-json -p dist`. Naming can be improved ;)